server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -302,7 +297,7 @@ private static String joinPaths(Iterable<Change<?>> changes) {
                          .thenCombine(historyFuture, (entryMap, commits) -> {
                              final ImmutableList<EntryDto<?>> entryDtos = entryDtos(repository, entryMap);
```

Comment:
We can remove this line :)

Reply:
And we don't have to do `repository.find()` here because we don't need it :)

Reply:
Oops! Fixed